List.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, TextInput, FlatList } from 'react-native';
```

Comment:
```suggestion
import { StyleSheet, View, FlatList } from 'react-native';
```
This file doesn't use Text or Textinput.